data/events/2022-lagos.yml

```
@@ -0,0 +1,94 @@
+#   variable: 2019-01-05T23:59:59+02:00
+# Note: we allow 2020-MM-DD for backward compatibility, but it can lead to unexpected behaviors (like your event disappearing from the front page during your last day)
+
+startdate: October 29th, 2022 # The start date of your event. Leave blank if you don't have a venue reserved yet.
+enddate: October 29th, 2022 # The end date of your event. Leave blank if you don't have a venue reserved yet.
+
+# Leave CFP dates blank if you don't know yet, or set all three at once.
+cfp_date_start: August 1st, 2022 # start accepting talk proposals.
+cfp_date_end: October 3rd, 2022 # close your call for proposals.
+cfp_date_announce: August 1st, 2022 # inform proposers of status
+
+cfp_link: "https://sessionize.com/localhost-conference-2022" #if you have a custom link for submitting proposals, add it here. This will control the Propose menu item as well as the "Propose" button.
+
+registration_date_start: August 1st, 2022 # start accepting registration. Leave blank if registration is not open yet
+registration_date_end: October 28th, 2022 # close registration. Leave blank if registration is not open yet.
+
+registration_closed: "" #set this to true if you need to manually close registration before your registration end date
+registration_link: "" # If you have a custom registration link, enter it here. This will control the Registration menu item as well as the "Register" button.
+
+# Location
+#
+coordinates: "6.54451°, 3.39226°" # The coordinates of your city. Get Latitude and Longitude of a Point: http://itouchmap.com/latlong.html
+location: "Lagos" # Defaults to city, but you can make it the venue name.
+#
+location_address: "Plot 9 Gbagada Industrial Scheme, Beside UPS, Gbagada Expressway" #Optional - use the street address of your venue. This will show up on the welcome page if set.
+
+nav_elements: # List of pages you want to show up in the navigation of your page.
+ # - name: propose
+ # - name: location
+ # - name: registration
+ # - name: program
+ # - name: speakers
+  - name: sponsor
+  - name: contact
+  - name: conduct
+# - name: example
+#   icon: "map-o" # This is a font-awesome icon that will display on small screens. Choose at http://fontawesome.io/icons/
+#   url: http://mycfp.com # The url setting is optional, and only if you want the navigation to link off-site
+
+
+# These are the same people you have on the mailing list and Slack channel.
+team_members: # Name is the only required field for team members.
+  - name: "Adedayo Akinpelu"
+    twitter: "@haroldsphinx"
+    employer: "Dapperlabs Inc"
+    linkedin: "https://linkedin.com/haroldsphinx"
+    github: "haroldsphinx"
+    image: "adedayo.jpg"
+  - name: "Idowu Emehinola"
+    employer: "Advanced Blockchain"
+    twitter: "@_hydeenoble"
+    github: "hydeenoble"
+    image: "idowu.jpg"
+  - name: AbdulBasit Kabir
+    employer: "Interswitch"
+
+organizer_email: "ga" # Put your organizer email address here
```

Not sure what `ga` is but this must be your mailing list.

```suggestion
organizer_email: "lagos@devopsdays.org" # Put your organizer email address here
```